deploy/dlt/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.15
+FROM python:3.11.11-alpine
```

Comment:
Note: switched to a python image, I can probably revert this if we want to use a vanilla alpine here.